### ISSUE TITLE
Why use `algorithm2e` package?

### ISSUE THREAD

**Issue by goretkin** [fmarotta/kaobook#177]
## Summary

https://github.com/fmarotta/kaobook/blob/ac3afaf53c748178a81cb9a1caff055711e4dfcb/kao.sty#L1321

## Details

It does not seem that this package is necessary for the kaobook style. Why not let users include it if necessary?

## Additional context

https://tex.stackexchange.com/questions/390683/why-should-you-avoid-using-complex-templates

**Comment by fmarotta** [comment]
Good point. I added it because I had found a customisation of the package options that fit well with the kao style, so that a user wouldn't need to re-discover these options in order to make the algorithm2e package blend well with the class. However, I'm now thinking of collecting all these unnecessary package into a new package called *kaoextra.sty*, to make the base class more 'lightweight'. Users can then choose whether to load or not the kaoextra package.
